website/docs/d/key_vault_key.html.markdown

```diff
@@ -59,6 +59,10 @@ The following attributes are exported:
 
 * `public_key_openssh` - The OpenSSH encoded public key of this Key Vault Key.
 
+* `resource_id` - The Azure control plane ID of the Key Vault Key.
+
+* `resource_versionless_id` - The Azure control plane Base ID of the Key Vault Key.
```

Comment:
I think this'd be more consistent as:

```suggestion
* `resource_id` - The (Versioned) ID for this Key Vault Key. This property points to a specific version of a Key Vault Key, as such using this won't auto-rotate values if used in other Azure Services.

* `resource_versionless_id` - The Versionless ID of the Key Vault Key. This property allows other Azure Services (that support it) to auto-rotate their value when the Key Vault Key is updated.
```

(we should also add a disclaimer about how each of these could be used, given they're special-purpose fields?)